HISTORY.rst

```diff
@@ -3,6 +3,85 @@
 Release History
 ---------------
 
+
+1.2.0 (2023-03-09) Taylor
+++++++++++++++++++++++++++
+
+**Special notes**
+
+New features:
+
+* Implement radius for NetworkGrid.get_neighbors #1564
+
+Some highlights for the perf improvements:
+
+* Use getattr for attribute strings in model data collection #1590 this is a 2x speedup over the relevant line, but not sure if compared to the rest of the function
+* Refactor iter/get_cell_list_contents methods #1570 at least 1.3x speedup for iter/get_cell_list_contents
+* Evaluate empties set more lazily #1546 (comment) ~1.3x speedup for place_agent, remove_agent, and move_agent
```

Comment:
Needs to add 1 more:
https://github.com/projectmesa/mesa/pull/1597 is for 1.3x speedup for grid access (`grid[x, y]`).